src/components/autocomplete/autocomplete.spec.js

```diff
@@ -1895,6 +1895,81 @@ describe('<md-autocomplete>', function() {
       expect(input[0].getAttribute('aria-activedescendant')).toBe('md-option-' + ctrl.id + '-1');
     });
 
+    it('should not retain the position of the scroll after panel is reopen', function() {
```

Comment:
```suggestion
    it('should not retain the position of the scroll after panel is reopened', function() {
```

Reply:
@Splaktar I just pushed the modified version